src/content/docs/tutorial-optimize-telemetry/detect-anomalies.mdx

```diff
@@ -0,0 +1,160 @@
+
+import omaoedgLookoutViewConsumingAccount from 'images/oma-oe-dg_screenshot-crop_lookout-view-consuming-account.webp'
+
+Once you've installed the [baseline dashboard](/docs/tutorial-optimize-telemetry/create-baseline-report/), you should keep an eye on it to detect any ingest anomalies as often as possible. While you have to wait until you have enough data to optimize your ingest, you can use your dashboard to find issues with your ingest much sooner. You can use the baseline dashbaord as is, or download optional dashboards to give you even greater anomaly detection options.
+
+You should also set up alerts to help notify you of issues as they occur. You can use our [ingest alerts guide](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts) to create a large variety of alerts, but we recommend you set up these two alerts as a minimum:
+
+* A threshold alert to notify if you exceed monthly targets for data ingest beyond seasonal increases
+* An anomaly alert to notify you of a sudden sharp increase ingest data
+
+<Callout variant="tip">
+New to creating alerts with our platform? Check out our [alert creation and management tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) to learn everything you need about creating New Relic alerts.
+</Callout>
+
+## Use the Lookout view to monitor anomalies [#lookout]
+
+The baseline dashboard you set up in the previous step of this tutorial has a feature called **Lookout**. In a nutshell, this feature lets you provide a NRQL query to instantly search for ingest anomalies over any period of time. For example, the following NRQL query will give you information on all ingest anomalies by `usageMetric` over the last 24 hours:
+
+```sql
+SELECT rate(sum(GigabytesIngested), 1 day) AS avgGbIngest FROM NrConsumption WHERE productLine='DataPlatform' FACET usageMetric 
+```
+
+        <img
+            src={omaoedgLookoutViewTelemType}
+            alt="Lookout view usage metric"
+            title="Lookout view usage metric"
+        />
+
+You can change the facet field to anything you want to get more, or less, granularity on your ingest data. For example, changing `usageMetric` to `consumingAccountName` would give you even more detailed information by reporting on ingest by account name, as shown below:
+
+        <img
+            src={omaoedgLookoutViewConsumingAccount}
+            alt="Lookout view consuming account"
+            title="Lookout view consuming account"
+        />
+
+### Install the entity breakdown dashboard (optional) [#install-entity-breakdown-dashboard]
+
+Previously in this tutorial, you installed the ingest baseline dashboard that uses `NrConsumption` as its primary source. But you may find that you also want information on estimating ingest traffic for other events or metrics. If you do, you can install the entity breakdown dashboard, which helps you create other visualizations that use `bytescountestimate()` to help estimate your ingest data. 
```

Comment:
I do this too and I think it's our writer-impulse to want to refer back to previous concepts like this. Is there an alternate structure we can use that streamlines this to be more linear without referring to 'previous' or 'before'?

Reply:
Sure can, easy enough fix to make. Thanks for the callout!